end-to-end-test/local/docker/setup_docker_containers.sh

```diff
@@ -0,0 +1,275 @@
+        --net=$DOCKER_NETWORK_NAME \
+        -v "$TEST_HOME/local/runtime-config/portal.properties:/cbioportal/portal.properties:ro" \
+        $backend_image_name \
+        sh -c 'cd /cbioportal/core/src/main/scripts; for FILE in /cbioportal/core/src/test/scripts/test_data/study_es_0/data_gene_panel_testpanel*.txt; do ./importGenePanel.pl --data $FILE; done'
+
+    # import study_es_0
+    docker run --rm \
+        --name=cbioportal-importer \
+        --net=$DOCKER_NETWORK_NAME \
+        -v "$TEST_HOME/local/runtime-config/portal.properties:/cbioportal/portal.properties:ro" \
+        $backend_image_name \
+        python3 /cbioportal/core/src/main/scripts/importer/metaImport.py \
+        --url_server "http://$E2E_CBIOPORTAL_HOST_NAME:8080/cbioportal" \
+        --study_directory /cbioportal/core/src/test/scripts/test_data/study_es_0 \
+        --override_warning
+
+    # import custom studies
+    for DIR in "$TEST_HOME"/studies/*/; do
+
+        docker run --rm \
+            --name=cbioportal-importer \
+            --net=$DOCKER_NETWORK_NAME \
+            -v "$TEST_HOME/local/runtime-config/portal.properties:/cbioportal/portal.properties:ro" \
+            -v "$DIR:/study:ro" \
+            $backend_image_name \
+            python3 /cbioportal/core/src/main/scripts/importer/metaImport.py \
+            --url_server "http://$E2E_CBIOPORTAL_HOST_NAME:8080/cbioportal" \
+            --study_directory /study \
+            --override_warning
+    done
+
+}
+
+check_jitpack_download_frontend() {
+    url_short="https://jitpack.io/com/github/$FRONTEND_PROJECT_USERNAME/cbioportal-frontend/$FRONTEND_SHA1_SHORT/cbioportal-frontend-$FRONTEND_SHA1_SHORT.jar"
+    sleeptime=0
+    maxtime=1200
+    while (($sleeptime < $maxtime)); do
+        if !(curl -s --head $url_short | head -n 1 | egrep "HTTP/[0-9.]+ 200"); then
+            echo "Waiting... ($url_short)"
+            sleep 10
+            sleeptime=$sleeptime+10
+        else
+            sleeptime=maxtime+1
+        fi
+    done
+
+    if !(curl -s --head $url_short | head -n 1 | egrep "HTTP/[0-9.]+ 200"); then
+        echo "Could not find frontend .jar (version: $FRONTEND_SHA1, org: $FRONTEND_PROJECT_USERNAME) at jitpack (url: $url_short)"
+        exit 1
+    fi
+}
+
+download_db_seed() {
+    # download db schema and seed data
+    curdir=$PWD
+    cd /tmp
+    curl $DB_CGDS_URL > cgds.sql
+    curl $DB_SEED_URL > seed.sql.gz
+    cd $curdir
+}
+
+run_session_service() {
+    docker stop mongoDB && docker rm mongoDB
+    docker run -d --name=mongoDB --net=$DOCKER_NETWORK_NAME \
+        -e MONGO_INITDB_DATABASE=session_service \
+        mongo:4.0
+
+    docker stop cbio-session-service && docker rm cbio-session-service
+    docker run -d --name=cbio-session-service --net=$DOCKER_NETWORK_NAME -p 8084:8080 \
+        -e JAVA_OPTS="-Dspring.data.mongodb.uri=mongodb://mongoDB:27017/session-service" \
+        thehyve/cbioportal-session-service:cbiov2.1.0
```

Comment:
prolly good to use a cbioportal/session-service image here as well. We might have to generate that. I don't think there's been an official one in a while. But we can also update that post merge